Library/Homebrew/cask/cask.rb

```diff
@@ -292,14 +292,20 @@ def artifacts_list
       artifacts.map do |artifact|
         next artifact.to_h if artifact.is_a? Artifact::AbstractFlightBlock
 
+        if artifact.is_a? Artifact::Relocated
```

Comment:
Are we sure that this should apply to all `Artifact::Relocated`? That would include the following artifacts.

* `Artifact::Artifact`
* `Artifact::Mdimporter`
* `Artifact::Moved`
* `Artifact::Prefpane`
* `Artifact::Qlplugin`
* `Artifact::Suite`
* `Artifact::Symlinked`
* `Artifact::Binary`

Reply:
Yes, that's the intention unless you can think of a case where this would not be desired